sqlglot/tokens.py

```diff
@@ -11,6 +12,14 @@
     from sqlglot.dialects.dialect import DialectType
 
 
+try:
+    import sqlglotrs  # noqa
+
+    USE_NATIVE_TOKENIZER = os.environ.get("SQLGLOT_NATIVE_TOKENIZER", "1") == "1"
```

Comment:
Does this mean we'll use the rust tokenizer by default?

Reply:
Yes, assuming a user installs the sqlglotrs package which would be an explicit action.